DefensiveItem.cpp

```diff
@@ -3,13 +3,10 @@
 #include "Paladin.h"
 #include "DragonSlayer.h"
 #include "Dragon.h"
```

Comment:
Are you using any of these in this cpp file?
- [ ] Don't Include What you Don't Use!!